test/controllers/media_controller_test.rb

```diff
@@ -0,0 +1,46 @@
+require 'test_helper'
+
+class MediaControllerTest < ActionController::TestCase
+
+  setup do
+    @media = media(:movie_all_fields)
+  end
+
+  test "should get index" do
+    get :index
+    assert_response :success
+    assert_not_nil assigns(@books)
```

Comment:
Good, you're checking that the page loads successfully and the instance variables are filled.